test/e2e-test-application/src/luigi-config/extended/settings.js

```diff
@@ -55,6 +55,56 @@ class Settings {
       }
     };
   }; */
+
+  userSettings = {
+    userSettingsNavigationEntry: {
+      label: 'My Settings',
+      icon: 'settings'
+    },
+    writeToCustomStorage: (key, obj) => {
+      sessionStorage.setItem(key, JSON.stringify(obj));
```

Comment:
Just curious, shall we use `localStorage` or `sessionStorage`?

Reply:
These functions are there if you don't want to use localstorage. You can implement these functions and implement your own logic to either use session storage or call some api to store data there.
If these functions are not implemented the localstorage will be used.